.gitignore

```diff
@@ -1,7 +1,7 @@
 __coverage__
 .build-info
 .sass-cache
-dist
+#dist
```

Comment:
@sushilshinde make sure to revert this line before merging ;)

Reply:
yes